torch/_inductor/ir.py

```diff
@@ -3725,6 +3725,21 @@ def create(cls, inputs, dim):
                     # use CL stride for the output
                     output_stride = make_channels_last_strides_for(new_size)
                     break
+        any_input_is_storage_and_layout = any(is_storage_and_layout(x) for x in inputs)
+        fx_node_args = V.graph.current_node.args[0]
+        if not isinstance(fx_node_args, (tuple, list)):
+            fx_node_args = [
+                fx_node_args,
+            ]
+        if any_input_is_storage_and_layout is False and any(
```

Comment:
suggest to add a note here explaining the heuristics we use here.

Reply:
Added, thanks!